scripts/verify-ai-claims.sh

```
@@ -97,12 +97,20 @@ echo "📁 Artifacts: $ARTIFACTS_DIR"
 echo ""
 
 # Activate virtualenv
-if [ -f venv/bin/activate ]; then
-    source venv/bin/activate
-elif [ -f .venv/bin/activate ]; then
-    source .venv/bin/activate
+if command -v uv &> /dev/null; then 
+    # Use uv if available
+    TEST_PREFIX = "uv run"
```

Shellcheck (0.11.0)

[error] Remove spaces around = to assign (or use [ ] to compare, or quote '=' if literal).

(SC2283)